database_post.php

```diff
@@ -3,8 +3,7 @@
 
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
try putting you connection to your DB in a seperate file, so this will optimise your code cause you are only connection once.

first inside a new php file called: dbConn.php put -->

function dbConn(){
    return new PDO('mysql:host=127.0.0.1;dbname=Portfolio','root');
}

Then in any file you call the $db -->
require '/dbConn.php'
$db = dbConn();